src/ts/legendControl.ts

```diff
@@ -79,24 +79,28 @@ module powerbi.extensibility.visual {
             legend.appendChild(titleElement)
 
             data.forEach(({colorStop, color}) => {
+                // Create line item
                 const item = d.createElement('div');
+
+                // Create color element and add to item
                 const colorElement = d.createElement('span');
                 colorElement.setAttribute("class", "mapbox-legend-color middle");
-                colorElement.setAttribute("style", "backgroundColor: color;");
+                colorElement.setAttribute("style", "background-color: " + color + ";");
```

Comment:
A less error-prone way of writing this is with template literals:
```
`background-color: ${color};`
```

Reply:
+1 for template literals